src/modules/binning.js

```diff
@@ -27,7 +27,7 @@ export function unBinResults(queryBinParams, results) {
         }
       // jscs:enable
       } else {
-        const intervalMs = TIME_LABEL_TO_SECS[timeBin] * 1000;
+        const intervalMs = timeBin === "auto" ? TIME_LABEL_TO_SECS["month"] : TIME_LABEL_TO_SECS[timeBin] * 1000;
```

Comment:
"auto" doesn't always result in "month". There should be function in immerse that determines what the "auto" interval is, should probably replace "auto" with the correct value in immerse instead of bleeding the logic into here crossfilter.